hobby-blog.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Julia Meier - Hobby Blog</title>
+  <link REL=StyleSheet HREF="stylesheets/normalize.css" TYPE="text/css" MEDIA=screen>
+  <link REL=StyleSheet HREF="stylesheets/styles.css" TYPE="text/css" MEDIA=screen>
+  <link href="https://fonts.googleapis.com/css?family=Bowlby+One+SC|Bungee+Inline|Bungee+Shade|Faster+One|Miltonian|Ribeye+Marrow" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Biryani:800|Bungee|Bungee+Inline|Montserrat+Alternates:700|Open+Sans:700" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Harmattan|Julius+Sans+One|Lekton|Linden+Hill|Pavanam|Rubik+Mono+One" rel="stylesheet">
+  <link href="https://afeld.github.io/emoji-css/emoji.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+  <header>
+    <h1><a href="index.html">Julia Meier</a></h1>
+    <nav>
+      <ul>
+        <li><a href="portfolio.html">Codery<a/></li>
+        <li><a href="hobby-blog.html">Hobby Blog<a/></li>
+        <li><a href="about.html">About Me<a/></li>
+      </ul>
+    </nav>
+  </header>
+
+  <body>
```

Comment:
not how the `<body>` tag works: <body> actually represents the idea of HTML that isn't the meta data (which goes in the <head> tag)

Read more here: http://www.htmldog.com/references/html/tags/body/

You make this error in a few of the pages